.github/workflows/bank-sdk.build.xcframeworks.yml

```diff
@@ -0,0 +1,101 @@
+name: Build BankSDK XCFrameworks
+on:
+  push:
```

Comment:
We need also `workflow_call` as a trigger in `on` to make this callable from another workflow. https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow

Reply:
like we have https://github.com/gini/gini-mobile-ios/blob/main/.github/workflows/bank-api-library.publish.docs.yml#L6